### DISCUSSION TITLE
Can't start Win11 arm installation on M1

### DISCUSSION THREAD

**Discussion by ansenkoy** [utmapp/UTM#5040] [UTM Help (macOS)]
I made several attempts with a few versions (e.g. 21390) for VMware, VirtualBox, and UTM. The last shot was with Windows11_InsiderPreview_Client_ARM64_en-us_25300.VHDX.
I've converted VHDX to qcow2 and tried to start the installation.
In all cases,
<img width="630" alt="Screenshot 2023-02-18 at 12 49 13" src="https://user-images.githubusercontent.com/10008429/219856526-e8dbd0de-75a9-40dc-8b2a-ef06cbbe0684.png">
 I've got the same result.
Please suggest what should I try next.

**Comment by baffourt** [comment]
Same result with m3, any help on this?

**Comment by Pfuschi64** [comment]
Just before this screen you see for a split second a kind of remark, saying to hit a key to start with dvd image or similar. Watch carefully to not miss this one and upon hitting it will start installing Windows.

**Comment by saulobmansur** [comment]
When this screen shows up, you can simply type "exit" to enter EFI settings and select a boot device.
